docs/src/main/sphinx/security/file-system-access-control.rst

```diff
@@ -465,29 +504,195 @@ name, and allow ``alice`` and ``bob`` to use a group principal named as
+information is denied. If no rule matches, system access is denied. Each rule is
+composed of the following fields:
 
 * ``user`` (optional): regex to match against user name. If matched, it
-  will grant or deny the authorization based on the value of ``allow``.
-* ``allow`` (required): set of access permissions granted to user. Values: ``read``, ``write``
+  grants or denies the authorization based on the value of ``allow``.
+* ``allow`` (required): set of access permissions granted to user. Values:
+  ``read``, ``write``
 
 For example, if you want to allow only the role ``admin`` to read and write
-system information, allow ``alice`` to read system information, and deny all other access, you
-can use the following rules:
+system information, allow ``alice`` to read system information, and deny all
+other access, you can use the following rules:
 
 .. literalinclude:: system-information-access.json
     :language: json
 
 A fixed user can be set for management interfaces using the ``management.user``
-configuration property.  When this is configured, system information rules must still be set
-to authorize this user to read or write to management information. The fixed management user
-only applies to HTTP by default. To enable the fixed user over HTTPS, set the
-``management.user.https-enabled`` configuration property.
+configuration property.  When this is configured, system information rules must
+still be set to authorize this user to read or write to management information.
+The fixed management user only applies to HTTP by default. To enable the fixed
+user over HTTPS, set the ``management.user.https-enabled`` configuration
+property.
+
+.. _catalog-file-based-access-control:
+
+Catalog-level access control files
+==================================
+
+You can create JSON files for individual catalogs that define authorization
+rules specific to that catalog. To enable catalog-level access control files,
+add a connector-specific catalog configuration property that sets the
+authorization type to ``FILE`` and the ``security.config-file`` catalog
+configuration property that specifies the JSON rules file.
+
+For example, the following Iceberg catalog configuration properties use the
+``rules.json`` file for catalog-level access control:
+
+.. code-block:: properties
+
+  iceberg.security=FILE
+  security.config-file=etc/catalog/rules.json
+
+
+Catalog-level access control files are supported on a per-connector basis, refer
+to the connector documentation for more information.
+
+.. note::
+
+    These rules do not apply to system-defined tables in the
+    ``information_schema`` schema.
+
+Configure a catalog rules file
+------------------------------
+
+Catalog-level rules JSON files are composed of three sections, each of which is
+a list of rules that are matched in order from top to bottom. The user is
+granted the privileges from the first matching rule. All regexes default to
+``.*`` if not specified.
+
+Schema rules
```

Comment:
None of these sections specify the name of the section to be used in the final JSON. We should fix this. No idea how it should look though.

Reply:
As a compromise, added some writing in the intro that shows the high level structure with the actual section names in code font.